pkg/ddevapp/snapshot.go

```diff
@@ -20,6 +20,11 @@ import (
 	"github.com/ddev/ddev/pkg/util"
 )
 
+type Snapshot struct {
+	Name    string
+	Created string
+}
```

Comment:
This type can probably be used for more things than just the list, but I took a brief look around and didn't see anywhere that seemed to actually need it right now.

Maybe it will be useful in https://github.com/ddev/ddev/issues/6045 somehow